README.md

```diff
@@ -1,221 +1,172 @@
+There are two simple ways to build the backend, for development and production use.
 
-#### Backend
+##### Building just the backend
 
-The backend is used to serve the frontend and as an API for getting files from the filesystem. The code is located at `frank-flow/main/java`.
+The easiest option while developing is to just run `mvn install`. This will only build the backend. This could be usefull if you're just developing the API or if the frontend is being served by the Frank!Runner.
 
 ##### Building the whole application
 
-There are two simple ways to build the backend, for local and server use.
-
-The easiest option while developing is to just run `mvn install`. This will build the backend and bundle existing front-end files into the Jar if present. While technically you could build the Jar without the frontend, there really isn’t a reason to do so. If you run `mvn inststall` multiple times, the front-end will be copied only the first time because `mvn install` does not clean up the target directory.
+The other option to run `mvn install -P frontend`. This command builds both the backend and the frontend. It is meant for production, because this command will also install Node.js and NPM to build the production frontend in a CI environment.
 
-The other option to run `mvn install -P frontend`. This command builds both the backend and the front-end. It is only meant for production, because with this command Maven will also install Node.js and NPM.
+### Configuration
 
-## Other projects
+The Frank!Flow has some configuration options which are mainly used for the backend. The configuration options can be set by placing them in a `frank-flow.properties` file. This file should be placed in the location from where the Frank!Flow gets start. While developing with the Frank!Runner this would be in the root of the folder `frank-runner`.
 
-- [Frank!Runner](https://github.com/ibissource/frank-runner)
-- [XSD code completion for the Monaco Editor](https://github.com/philipsens/monaco-xsd-code-completion)
+| Property                   | Description                                      | Example                                                      |
+| -------------------------- | ------------------------------------------------ | ------------------------------------------------------------ |
+| `frank-flow.frontend-path` | The location of the dist folder for the frontend | `frank-flow.frontend-path = /home/user/Projects/frank-flow/frank-flow/target/frontend` |
+| `frank-flow.context-path`  | The URL at which the Frank!Flow will be served   | `frank-flow.context-path = /frank-flow/`                     |
+| `frank-flow.port`          | The port which the build-in Tomcat should use    | `frank-flow.port = 8080`                                     |
+| `frank-flow.war`           | The webapp war                                   | `frank-flow.war = /frank-flow-webapp.war`                    |
```

Comment:
The "=" signs may be confusing, for example in: `frank-flow.war = /frank-flow-webapp.war`. I guess you are giving the default value, but I did not see so immediately. Can you make this more clear? Maybe you can explain right before or right after the table. The advantage of the "="-syntax here is that it is short.

Reply:
Ok.